pkg/download/version_zip.go

```diff
@@ -44,7 +44,10 @@ func ZipHandler(dp Protocol, lggr log.Entry, df *mode.DownloadFile) http.Handler
 		defer zip.Close()
 
 		w.Header().Set("Content-Type", "application/zip")
-		w.Header().Set("Content-Length", strconv.FormatInt(zip.Size(), 10))
+		size := zip.Size()
+		if size >= 0 {
```

Comment:
Can we change this to `if size > 0` (instead of >=) so that we don't have to set `-1` on every storage implementation? 

There's very little to 0 percent chance that we have a proper module that happens to be 0 bytes...so I think it's safe to assume that a 0 size is just an "unknown size" as opposed to an actually 0 byte zip file.